.gitignore

```diff
@@ -19,4 +19,7 @@ Thumbs.db
 # Preview artifacts #
 #####################
 _pelican
-_output
\ No newline at end of file
+_output
+
+# environment
+env/
```

Comment:
I suggest reverting this addition. Given that people have different preferences regarding virtual environment locations, I think it would be better to put this kind of entry on individual workstations in `~/.gitignore` instead of inside the project-wide `.gitignore` file.